statistic-service/src/main/java/com/company/statistic/controllers/StatisticController.java

```diff
@@ -0,0 +1,19 @@
+package com.company.statistic.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("api/v1/statistic")
+public class StatisticController {
+
+    @GetMapping("/{id}")
+    ResponseEntity<Float> getRating(@PathVariable UUID userId) {
```

Comment:
Not so important right now. But eating if what? And why it's float?

Reply:
Rating for the last 50 games for example. I thought to make the rating float, but it will probably look better with an integer